src/transaction/log_tran_table.c

```diff
@@ -1457,6 +1458,28 @@ xlogtb_dump_trantable (THREAD_ENTRY * thread_p, FILE * out_fp)
   fprintf (out_fp, "\n");
 }
 
+/*
+ * logtb_get_tran_memory_info_nolatch - get the transaction memory information
+ *                                      with nolatch
+ *
+ * return: nothing
+ *
+ */
+void
+logtb_get_tran_memory_info_nolatch (std::vector < std::pair < int, uint64_t >> &tran_info)
+{
+  LOG_TDES *tdes;
+
+  for (int i = 0; i < NUM_TOTAL_TRAN_INDICES; i++)
+    {
+      tdes = log_Gl.trantable.all_tdes[i];
+      if (tdes != NULL && tdes->trid != NULL_TRANID && tdes->state == TRAN_ACTIVE)
```

Comment:
After COMMIT/ABORT, tdes could have same condition. (Need to check)
- log_complete() -> logtb_clear_tdes()

Reply:
Upon checking the mentioned content, it appears that the state of transactions is generally being properly managed during commit, but there seem to be some vulnerabilities during checkpoint situations or certain system transaction processes. Therefore, merely checking `tdes->state == TRAN_ACTIVE` is not sufficient to determine if the transaction is currently active. However, it is believed that this can be overcome by checking `tdes->trid != NULL_TRANID`, as confirmed in the dump_trantable() function, where output is determined based on such conditions.